src/components/DoDontExample/DoDontExample.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import { CheckmarkFilled24, CloseFilled16 } from '@carbon/icons-react';
+import CheckmarkFilled24 from '@carbon/icons-react/es/CheckmarkFilled/24';
+import CloseFilled16 from '@carbon/icons-react/es/CloseFilled/16';
```

Comment:
`import CloseFilled16 from '@carbon/icons-react/es/CloseFilled/16';`

should be
`import misuse20 from '@carbon/icons-react/es/misuse/20';`

Where would the part about styling the `inner-path` for these icons be in this context?